src/accordion/Accordion.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @overview An Accordion module comprised of sections that toggle open and close
+ * @module Accordion.js
+*/
+
+export const Accordion = {
+
+  accordionSection : `.js-accordion-section`,
+  accordionSectionTrigger: `.js-accordion-section-toggle`,
+  accordionSectionContent: `.js-accordion-section-content`,
+  openClass: `is-open`,
+
+  /**
+   * Inits accordion elements
+   * @returns {void}
+  */
+  init() {
+    this.accordionSections  = document.querySelectorAll(this.accordionSection);
+
+    [...this.accordionSections].forEach((el) => {
+      let trigger = el.querySelectorAll(this.accordionSectionTrigger);
+      trigger = trigger[0];
+      let content = el.querySelectorAll(this.accordionSectionContent);
+      content = content[0];
+
+      trigger.setAttribute(`touch-action`, `none`);
+      content.setAttribute(`aria-expanded`, false)
+
+      trigger.addEventListener('pointerup', (e) => {
+        e.stopPropagation();
+
+        if (content.classList.contains(this.openClass)) {
+          this.closeAccordionSection(content);
+        }
```

Comment:
Do we have a convention to have else on the same line as closing if line? eg `} else {`